doc/api/os.md

```diff
@@ -441,6 +441,8 @@ added: v6.0.0
     values will be `Buffer` instances. **Default:** `'utf8'`.
 * Returns: {Object}
 
+Throws an `SystemError` when user has no `username` or `homedir`.
```

Comment:
I don't think this should be the first thing in the doc about the function. This information should come after the description of what the function actually does.

```suggestion
Throws a `SystemError` if user has no `username` or `homedir`.
```